### ISSUE TITLE
Canales en Django

### ISSUE THREAD

**Issue by lexotero** [DjangoMadrid/intereses#2]

**Comment by RusEu** [comment]
+1

**Comment by yamila-moreno** [comment]
+1 aunque creo que para el día 6 yo no podré asistir #ay

**Comment by munikes** [comment]
+1

**Comment by Dieg0x17** [comment]
+1 
Me interesa el hecho de usar websockets y peticiones http2 integrados con el core de django directamente y no a través de celery u otras alternativas, pero channels todavía es joven así que mejor dejarlo para futuras charlas, desde mi humilde punto de vista prefiero que la de marzo sea de deploy.

**Comment by lexotero** [comment]
Estoy de acuerdo con @Dieg0x17, aunque tampoco es tan joven... En la siguiente versión de Django (que caerá en verano) ya será parte del core.

**Comment by tapia** [comment]
+1
Aunque aún no ha entrado en el core, hasta donde yo sé el API está ya definido y estable, así que yo por lo menos tengo ganas de ir empezando a jugar con ello.
